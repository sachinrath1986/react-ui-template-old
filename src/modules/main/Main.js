import Aside from "./aside/Aside"

const Main = () => {

    return (
        <div className="d-flex flex-column flex-root">
            <div className="d-flex flex-row flex-column-fluid page">
                <Aside />
                I am dashboard
                </div>
        </div>
    );
}

export default Main;