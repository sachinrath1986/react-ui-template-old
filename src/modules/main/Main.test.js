import { render } from "@testing-library/react";
import Main from "./Main";

jest.mock("./aside/Aside", () => () => <div>Mock Aside component</div>);

describe("Main", () => {
  it("renders snapshots", () => {
    const { container } = render(<Main />);
    expect(container).toMatchSnapshot();
  });
});
