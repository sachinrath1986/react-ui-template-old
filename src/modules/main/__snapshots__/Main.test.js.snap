// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders snapshots 1`] = `
<div>
  <div
    class="d-flex flex-column flex-root"
  >
    <div
      class="d-flex flex-row flex-column-fluid page"
    >
      <div>
        Mock Aside component
      </div>
      I am dashboard
    </div>
  </div>
</div>
`;
